test/cat.jl

```diff
@@ -154,23 +154,23 @@ module TestCat
         @test typeof.(dt.columns) == [Vector{Int}]
         dt = vcat(DataTable([CategoricalArray([1])], [:x]), DataTable([[1]], [:x]))
         @test dt == DataTable([CategoricalArray([1, 1])], [:x])
-        @test typeof.(dt.columns) == [CategoricalVector{Int, drf}]
+        @test typeof.(dt.columns) == [CategoricalVector{Int, DRT, Int, Union{}}]
```

Comment:
I'd rather rely as little as possible on the two last type parameters, as the implementation might change in the future. Could you check only the two first ones, using `<:`? Same below.